src/main/kotlin/com/rose/gateway/minecraft/logging/FileLogger.kt

```diff
@@ -0,0 +1,28 @@
+package com.rose.gateway.minecraft.logging
+
+import com.rose.gateway.GatewayPlugin
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+import java.util.logging.FileHandler
+import java.util.logging.Logger
+
+/**
+ * Helper object that provides logging to file
+ */
+object FileLogger : KoinComponent, Logger("File Logger", null) {
+    /**
+     * @property logger this is the logger as a var
```

Comment:
You can drop the "@property logger" here because this comment is directly on the logger property